trackers/browser-tracker/test/tab_id_storage.test.ts

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
I'm a bit confused about the placement of this file, is it intentional that it is in `trackers/browser-tracker`? The feature is implemented in `libraries/browser-tracker-core` so I would expect the tests to be there, but maybe I'm missing some requirement here.

Reply:
Actually this took a bit of my time to think about too. This was my train of thought that resulted in my adding it there:
- The tests on `libraries/browser-tracker-core` are about internal implementations of the tracker system. _(E.g. `newTracker|addTracker|Tracker` is never used in these tests.)_
- The tests on `trackers/browser-tracker` had already tests that described high level behaviour.
Based on the above points and the thinking that this is a high level feature to begin with, I added the tests there.
WDYT ?

Reply:
So these are kind of like in between unit and integration tests? I'm fine with that and it makes sense to also test on this level, but it leaves the `trackers/javascript-tracker` without testing the functionality. We can keep it like this, just something about it is bugging me that we should find a way to distribute the test coverage to multiple levels.

Reply:
Will discuss it internally as soon as we get the chance and we will come up with the correct strategy.